### ISSUE TITLE
ArrowLoader behind 0.15.1 release

### ISSUE THREAD

**Issue by manzt** [uber-web/loaders.gl#531]
I believe there have been some breaking changes since the 0.13 release of Arrow, and the loader cannot read a new file with the 0.15.1 format. 

Example:
```javascript
import { ArrowLoader } from "@loaders.gl/arrow";
import { load } from "@loaders.gl/core";

const ARROW_DATA_URL = 
  "https://gist.githubusercontent.com/manzt/" +
  "ad8c2fe8063b03461b5480ba45f292d6/raw/" +
  "ef89111fe3c352913f277397150d6226e344b29d/" +
  "linnarsson.molecules.arrow";

const data = await load(ARROW_DATA_URL, ArrowLoader);
console.log(data);
// Object { name: [], x: Uint16Array [], y: Uint16Array [], __index_level_0__: Int32Array [] }
```

**Comment by manzt** [comment]
Working example with arrow 0.15.1:
https://observablehq.com/@manzt/vitessce-loading-time

**Comment by ibgreen** [comment]
Published in 2.1.0
